Backend-API/src/main/java/com/avangers/backendapi/services/UserService.java

```diff
@@ -5,4 +5,5 @@
 
 public interface UserService {
     ResponseEntity<String> addUser(RegisterUserDTO registerUserDTO);
+    ResponseEntity<String> updateUser(Long userId, RegisterUserDTO registerUserDTO);
```

Comment:
We probably do not want `ResponseEntity`s here anymore.

Reply:
If I undestand that corectly after some googling we want that UserConstroller will handle Http.